Recipe/recipe.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title> My Recipe Page</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Sarah's Recipe Page</title>
+  <link href="./recipe.css" rel="stylesheet" type="text/css" />
+  
+
+
+  <head>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Teko|Abril Fatface|Yeseva One|Oleo Script|Tangerine|Assistant">
+    <style>
+    h1 {font-family: "Oleo Script", serif; font-size: 62px; text-shadow: 4px 4px 4px rgb(124, 152, 123); color: rgb(34, 77, 31)}
+    h2 {font-family: "Assistant", serif; font-size: 25px; color: rgb(0, 0, 0)}
+    </style>
+    </head>
+
+
+  
+</head>
+
+<body>
+  
+ <h1 class="center" style="margin-bottom: 0"><b>Imbaattan Recipe</b></h1> 
+ <h2 class="center"> (Fried Potato Wedges with Minced Meat) </h2>
+  
+
+  
+<div class="recipe-image" id="specialdiv">
+<img src="https://arousingappetites.com/wp-content/uploads/2022/08/MBattan.jpeg?ezimgfmt=ng%3Awebp%2Fngcb1%2Frs%3Adevice%2Frscb1-1" width="250" height="200" class="center" style="margin-bottom: 50px">
+</div>
+<p class="center"><b>Description:</b><br>This delicious deep fried finger food originates and is unique to Libya,<br> my home country in North Africa! <br> It's especially popular during special occasions as well as Ramadan.<br> The well seasoned flavours of the meat and the combination <br> of the crispy exterior of the potatoes might <br> make this your new favourite side dish! <br> North African recipes are passed down generationally and typically <br> don't have exact measurments. <br> Below is the closest appoximation of measurments <br> to the recipe my mother makes.<p class="center">Enjoy and Sahtain! (Arabic word for Bon Appetit!) </p>   </p>
+<img src="https://www.worldatlas.com/r/w960-q80/upload/91/07/a8/shutterstock-751043653.jpg" width="260" height="210" class="center">
+
+<div>
+    <h4 class="center">Meat mixture:</h4>
+    <ul class="centerforText">
+        <li>1.5 lb ground beef</li>
+        <li>5-6 large potatoes</li>
+        <li>1 cup finely chopped parlsey</li>
+        <li>1/2 cup finely chopped cilantro</li>
+        <li>1 medium onion finely chopped</li>
+        <li>2 cloves garlic grated</li>
+        <li>1 tablespoon salt</li>
+        <li>2 egg whites</li>
+        <li>Preferably use Libyan Spice Mix<br> if available otherwise add:</li>
+        <li>1/4 teaspoon black pepper</li>
+        <li>1/2 teaspoon ginger</li>
+        <li>a few pinches of cinnamon</li>
+        <li>1/4 teaspoon tumeric</li>
+        <h4 class="center">Coating:</h4>
```

Comment:
The only element that's valid inside of `<ul>` or `<ol>` is the `<li>` tag.

See https://github.com/sarahtech2022/TechtonicaAssignments/pull/1#discussion_r1084783321